config/middlewares.js

```diff
@@ -78,11 +78,22 @@ module.exports = [
       sameSite: process.env.NODE_ENV === "development" ? 'Lax' : 'None',
     },
   },
-  'strapi::poweredBy',
-  'strapi::logger',
-  'strapi::query',
-  'strapi::body',
-  'strapi::favicon',
-  'strapi::public',
-  'plugin::users-permissions.jwt',
+
+  "strapi::poweredBy",
+  "strapi::logger",
+  "strapi::query",
+  //"strapi::body",
```

Comment:
We can remove this because you are customizing the same plugin in https://github.com/dev-launchers/strapiv4/pull/250/files#diff-612406ce62e8d660b3765c04a53ca19f08edfec7a3539b6b12b942f3e509636dR89.